bigframes/session/__init__.py

```diff
@@ -1496,23 +1513,24 @@ def read_gbq_function(
             session=self,
         )
 
-    def _start_query(
-        self,
+    @staticmethod
```

Comment:
staticmethod should be avoided. https://g3doc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head

Reply:
Ack. refactored in the recent patch, PTAL.